src/db/migration.003.sql

```diff
@@ -0,0 +1,22 @@
+BEGIN;
```

Comment:
Please follow the conventions from the other SQL code.

- PascalCase for table names, I think
- and snakeCase for column names
- whitespace after `,`
- line breaks and indentation in the queries
- line break at file end